ui/StatusQ/src/StatusQ/Core/LocaleUtils.qml

```diff
@@ -236,9 +236,16 @@ QtObject {
 
     // return full days between 2 dates
     function daysBetween(firstDate, secondDate) {
```

Comment:
You're changing the semantics of an existing function... don't like that :)

Better keep this one intact and create a new one instead